src/framework/visualization/retrieveIntegrationData.ts

```diff
@@ -0,0 +1,45 @@
+import globby from 'globby';
+import upath from 'upath';
+
+import { Relationship, Entity } from '../types';
+import { readJsonFromPath } from '../../fileSystem';
+import { IntegrationData } from './types/IntegrationData';
+import { IntegrationMissingCollectJSON } from './error';
+
+/**
+ * Retrieve integration data from the JSON files from the cache directory
+ */
+export async function retrieveIntegrationData(
+  integrationPath: string,
+): Promise<IntegrationData> {
+  const [entityPaths, relationshipPaths] = await Promise.all([
+    globby([upath.toUnix(`${integrationPath}/index/entities/**/*.json`)]),
+    globby([upath.toUnix(`${integrationPath}/index/relationships/**/*.json`)]),
```

Comment:
ATM the `.j1-integration/index` directory only contains symlinks to files in the `.j1-integration/graph` directory and is grouped by `_type`, but this could potentially change in the future if we decide to add support for grouping by other properties. 

Also, one thing we are planning for with this sdk is for developers writing integrations in other languages to use the CLI to perform synchronization with our persister (see [here](https://github.com/JupiterOne/integration-sdk/blob/master/docs/development.md#keeping-the-door-open-for-developers-not-using-javascript-or-typescript)). Those developers my not even end up having an `index` directory. 

The goal was to just allow those developers to just write to the `.j1-integration/graph` with directory structure they want and still be able to perform synchronization. The loose requirements allows for developers to not worry about matching our own structure since they may end up coming up with a different structure that suites them better (they may not have the concept of steps). It also allows for people to just write json to the `.j1-integration/graph` directory by hand for the _super_ simple integrations where devs just want to use j1 as a simple config store. I'd like to still allow for visualizations to work for these developers.

Reply:
Got it, so it is better to just glob the graph directory instead?

Reply:
Yep!